workflows/scripts/feature_select.py

```diff
@@ -85,7 +72,7 @@ def feature_selection(
 if __name__ == "__main__":
     all_norm_profile = [str(f_in) for f_in in snakemake.input]
     out_files = [str(f_out) for f_out in snakemake.output]
-    config_path = str(snakemake.params["feature_select_config"])
+    config_path = snakemake.params["feature_select_config"]
```

Comment:
Noticing the difference here from `workflows/scripts/convert.py`, consider keeping the approach with files under `workflows/scripts/...` consistent with one another to help reduce the overall complexity of maintaining these.

Reply:
Definitely, I wrote an issue in regards of the inconsistency. [#40](https://github.com/WayScience/CytoSnake/issues/40)